public/assets/img/ecosystem/Dapp.tsx

```diff
@@ -1,32 +1,50 @@
-import Image from "next/image";
-import WalletAppsIcon from "./wallet-apps.png";
-import WalletAppsIconHover from "./wallet-apps-hover.png";
+import { Video } from "@components/commons/Video";
 
 interface WalletAppsProps {
   id: string;
-  className?: string;
 }
 
 export function WalletApps(props: WalletAppsProps): JSX.Element {
-  const { id, className } = props;
+  const { id } = props;
+
   return (
-    <Image
-      src={WalletAppsIcon}
-      alt="Wallet Apps Icon"
-      id={id}
-      className={className}
-    />
+    <div
+      style={{
+        position: "absolute",
+        top: "-175px",
+        right: "-65px",
+      }}
+    >
+      <Video
+        poster=""
+        src="/assets/img/ecosystem/dapp.mov"
+        width={500}
+        height={500}
+        id={id}
+      />
+    </div>
   );
 }
 
 export function WalletAppsHover(props: WalletAppsProps): JSX.Element {
-  const { id, className } = props;
+  const { id } = props;
+
   return (
-    <Image
-      src={WalletAppsIconHover}
-      alt="Wallet Apps Icon"
-      id={id}
-      className={className}
-    />
+    <div
+      style={{
+        position: "absolute",
+        top: "-175px",
+        right: "-65px",
+        bottom: "-200px",
+      }}
+    >
+      <Video
+        id={id}
+        poster=""
```

Comment:
can we add a poster here?

Reply:
Can but because the poster is the first thing that the card will load before the video, especially on hover, it will look distorted

Reply:
do you have a screenshot of how it looks like? is it because of the size of the poster?

Reply:
https://user-images.githubusercontent.com/44501120/227138723-ee0af92b-a0bc-4c07-b781-80178da9a867.mov

will ask the designers for the exact dimension so that we can try to layer ontop of the video nicely

Reply:
ok nvm let's leave it.

Reply:
Removed!

Reply:
Removed!